src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/utility/Constants.java

```diff
@@ -1,4 +1,11 @@
 package uk.gov.ons.ctp.integration.contactcentresvc.utility;
 
 /** Constants used across the service */
-public class Constants {}
+public class Constants {
+
+  /**
+   * This value is used by the case service in the 'sampleUnitType' field of a Case response to
+   * represent a household.
+   */
+  public static String CASE_SERVICE_UNIT_TYPE_HOUSEHOLD = "H";
```

Comment:
We already have an enum CaseType in the service's API representation package

Reply:
Code updated to use CaseType. 
Deleted the previously added constant.